sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -1008,4 +1008,45 @@ class SparkSqlParserSuite extends AnalysisTest with SharedSparkSession {
           stop = sql.length - 1))
     }
   }
+
```

Comment:
Do we have a test coverage with `LEGACY_KEEP_COMMAND_OUTPUT_SCHEMA` somewhere else?

Reply:
Since we are touching it, can we follow the other commands and add a new `ShowNamespacesParserSuite`? an example is `DropTableParserSuite`